src/SFA.DAS.ApplyService.Web.UnitTests/Controllers/RoatpApplicationControllerTests.cs

```diff
@@ -360,9 +360,7 @@ public void Confirm_submit_application_updates_application_status_and_sends_conf
                 }
             };
 
-            _apiClient.Setup(x => x.GetOrganisationByUserId(It.IsAny<Guid>())).ReturnsAsync(organisationDetails);
```

Comment:
Is this supposed to be removed? I couldn't find the corresponding line removed in RoatpApplicationController

Reply:
No I mistook it for a call buried in the GetApplications handler, I'll reinstate